content/en/observability_pipelines/sources/_index.md

```diff
@@ -54,9 +54,46 @@ After events are ingested by the source, they get sent to different processors a
 
 **Note**: The `bytes in per second` metric in the UI is for ingested raw events, not enriched events.
 
+## TLS certificates
+
+Enable TLS for Observability Pipelines to ensure that logs are encrypted during transit and prevent attackers from tampering with your log data.
+
+Observability Pipelines does not accept self-signed certificates by default because self-signed certificates do not provide secure trust verification and can potentially expose your environment to man-in-the-middle attacks.
```

Comment:
```suggestion
Observability Pipelines does not accept self-signed certificates by default because they do not provide secure trust verification and can potentially expose your environment to man-in-the-middle attacks.
```